### DISCUSSION TITLE
Recognize the table and include the title in the table

### DISCUSSION THREAD

**Discussion by Lihwnlp** [jsvine/pdfplumber#1236] [Ask for help with specific PDFs]
Very helpful project！！
I have a question: when I use .extract_tables to extract tables, the table's title and the text below the table are also being recognized. How should I configure it? 
My form looks like this:
![Screenshot 2024-12-09 135449](https://github.com/user-attachments/assets/6ffb59da-c033-46c1-a345-1363c56949fb)
The extracted table looks like this；
![image](https://github.com/user-attachments/assets/d2149213-1feb-49eb-927b-c921e2cd0422)
[[[‘Table 2: abcdefg, None,None,None],
[‘ss', ‘ss’, ‘ss', ‘ss’],
[‘ss', ‘ss’, ‘ss', ‘ss’],
[‘ss', ‘ss’, ‘ss', ‘ss’],
[‘ss', ‘ss’, ‘ss', ‘ss’],
['4.9. cwniocnoiwenciewnciowenciewnc,None,None,None,None]]]

**Comment by jsvine** [comment]
Hi @Lihwnlp, can you attach the PDF in question? Without it, it will be difficult to propose a specific solution for your specific situation. Based on the screenshots, however, it seems that there are likely some non-visible graphical elements being picked up (perhaps because they are white-on-white). You can check whether this is the case by running `page.to_image().debug_tablefinder()`.

You can also examine the `.rects`, `.lines`, and `.curves` object lists to try to figure out which specific objects are causing the issue, and then use `page.filter(...)` to exclude them. (Searching for `filter` in the discussions should point you to solutions for similar situations.)
